doc/api/modules.md

```diff
@@ -200,10 +200,12 @@ Modules are cached after the first time they are loaded. This means
 (among other things) that every call to `require('foo')` will get
 exactly the same object returned, if it would resolve to the same file.
 
-Multiple calls to `require('foo')` may not cause the module code to be
-executed multiple times.  This is an important feature. With it,
-"partially done" objects can be returned, thus allowing transitive
-dependencies to be loaded even when they would cause cycles.
+By default (provided the `require.cache` is not modified), multiple calls to 
+`require('foo')` will not cause the module code to be executed multiple times.  
+
+This is an important feature. With it, "partially done" objects can be returned, 
```

Comment:
This line has trailing space and exceeds the limit of 80 characters producing linter issue.